js/packages/components/dropdowns/Dropdown.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import { storiesOf } from '@storybook/react-native'
+import Long from 'long'
+import React, { useState } from 'react'
+import { ScrollView, Text, View } from 'react-native'
+
+import beapi from '@berty/api'
+
+import { TextualDropdown, MembersDropdown } from '.'
+import { AppDecorator } from '../../../.storybook/preview'
+
+const Spacer = () => <View style={{ height: 16 }} />
+
+const members: beapi.messenger.IMember[] = [
```

Comment:
Now with the mock, i think you can get it from the mocked datas and not redefine it like that 👍

Reply:
1. You can just go to the mocked datas and see it
2. In our mocked datas you have a lot of different scenarios in every data points
3. I think the dependence to Redux is not a reason, because its more related on how we design our components implementation than use Redux or not, because its the only component that use redux if i correctly remember, so maybe it was not correctly implemented, or its a more complexe component
4. For sure you don't have development freedom

Honestly as you want, maybe you can implement both and comment the redux one with a comment explain that you can use the redux one if you want to have a lot of datas in some scenarios, and if you want very specific scenarios you can use `custom datas` 👍